plugins/alembic/module/vtkF3DAlembicReader.cxx

```diff
@@ -20,51 +22,334 @@
 #pragma warning(pop)
 #endif
 
+using IndicesContainer = std::vector<int32_t>;
+using V3fContainer = std::vector<Alembic::Abc::V3f>;
+using AttributesContainer = std::map<std::string, V3fContainer>;
+using PerFaceWavefrontIndicesTripletsContainer = std::vector<Alembic::Abc::V3i>;
+using PerMeshWavefrontIndicesTripletsContainer =
+  std::vector<PerFaceWavefrontIndicesTripletsContainer>;
+
+const size_t P_indices_offset = 0;
```

Comment:
Our coding rule is "camelCase" for variables, starting an uppercase for class members, and lowercase otherwise.